src/hooks/useDebouncedTrade.test.ts

```diff
@@ -0,0 +1,72 @@
+import { renderHook } from '@testing-library/react'
```

Comment:
this file is the almost the same minus the client side hook tests and the filename